entrypoint.sh

```diff
@@ -191,7 +191,11 @@ function execute_push {
     extra="${options["extra"]}"
   }
 
-  local request="cloudsmith push ${options["action"]} ${options["format"]} $context ${options["file"]} $params $extra $tags"
+  local executable="cloudsmith"
+  if [ "$USE_EXECUTABLE" = "true" ]; then
+    executable="${GITHUB_ACTION_PATH}/cloudsmith"
+  fi
+  local request="$executable push ${options["action"]} ${options["format"]} $context ${options["file"]} $params $extra $tags"
```

Comment:
is any of this section necessary if the executable has the same name in both cases?

Reply:
Fixing in new PR